### PR TITLE
Canonical MUR page (archive MURs)

### PR DESCRIPTION
**Pull request by adborden**
Adds view for the canonical MUR page (for archived MURs). Resolves https://github.com/18F/fec-eregs/issues/227
- [ ] https://github.com/18F/fec-style/pull/510

@jenniferthibault since both archived and current MURs will have a name field, I moved the archive case indicator under the summary, but not really sure where it should go.

screenshot from 2016-09-15 14-37-23

### CODE REVIEW

File: openfecwebapp/templates/legal-mur.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/breadcrumbs.html' as breadcrumb %}
+{% import 'macros/legal.html' as legal %}
+
+{% set breadcrumb_links=[(cms_url + '/legal-resources', 'Legal resources'), (url_for('advisory_opinions_landing'), 'Enforcement')] %}
```
Comment (adborden) [inline openfecwebapp/templates/legal-mur.html]: No landing page for Enforcement Matters yet, so this is just a placeholder.

**Comment by jenniferthibault** [comment]
Hi @adborden !

Sorry, it looks like the issue you were building from in https://github.com/18F/fec-eregs/issues/227 didn't get the most recent design screenshot. :head-desk: BUT it's not very far off at all! 

In the design issue https://github.com/18F/fec-eregs/issues/244#issuecomment-244986834 I had put the "Archived" text under the name field, like this:

screen shot 2016-09-16 at 10 18 33 am

Looking at it again, I like your suggestion of putting it in line with the content, because it doesn't crowd the title space as much, but it needs a little more spacing, and the underline under "Summary" to help delineate the space. Like this:

screen shot 2016-09-16 at 10 40 01 am


**A few design notes:**
(I don't have the legal side set up for running locally, so this is just based off of your screenshot)
- Add underline under "Summary" (for both Archived & Current MURs)
 - screen shot 2016-09-16 at 10 45 08 am
- For the sidebar, I was _nearly certain_ that we had used this headline style pattern before and that I wasn't inventing something new, but now I can't find it! The closest I can find to show you what I was trying to go for is [this sidebar](https://beta.fec.gov/registration-and-reporting/get-treasurer/). Since all of the other styles we use in the sidebar are sans-serif, I'd like to keep it in that family, and not break out to the default serif that `h5` uses. So it looks like this isn't a new _style_, (because it's the same bold sans serif style we use for data table column titles) just a new place for it? What I'm saying is, I'd like the headlines in the sidebars to be the same style as data table column titles, but I don't know how to build the HTML structure intelligently, (and I'm hoping you have ideas!) 😁  
  - screen shot 2016-09-16 at 10 52 54 am
- Could we add some space between the right edge of the "View Case" button, and the document size text? The Sketch file has 15px looking 👌 
  - 
screen shot 2016-09-16 at 11 28 00 am

Since I don't have this running locally, could you share a screenshot of the mobile view, and the participants table?

Or maybe it's time I set things up to get it running locally 💭 ...

**Comment by jenniferthibault** [comment]
Hold on a hot sec! I found where that sidebar typography pattern style came from in my design files, and why I thought it existed already!

We are intending to use it on the [canonical AO page](https://beta.fec.gov/data/legal/advisory-opinions/2009-22/), but that's still listed on site as "Coming soon". So it's not unique.

screen shot 2016-09-16 at 12 02 52 pm

**Comment by adborden** [comment]
Okay, no worries about the mockups being mixed up. I thought something was fishy and should've asked. All comments sound good. The sidebar comment is a little confusing to me, so I might grab you later.

To get setup locally, try this:

    FEC_FEATURE_LEGAL_MURS=1 FEC_WEB_API_URL=https://fec-dev-api.18f.gov python manage.py runserver

Ping me if you run into any problems, I'm happy to pair!

**Comment by adborden** [comment]
@anthonygarvan do you mind doing the tech review on this since you're looking at it anyway?

**Comment by adborden** [comment]
@jenniferthibault can you take another look? I made some tweaks from our pairing on Friday but forgot to write them down. Hopefully I got everything:

- Add rule to Summary header
- Include extra margin on archive case
- Extra right spacing on View case button
- Remove margin between Participants header and table
- Add vertical rules to table
- Add Top of page link for mobile

**Comment by adborden** [comment]
Oh @jenniferthibault I remembered one more, only show the citations sidebar if the citations exist. Looking for one to test, I came across this one which is pretty empty. Not sure how to handle it. I might follow this one up in a bug though, don't think this is particularly common.

http://localhost:3000/legal/matter-under-review/3584/

screenshot from 2016-09-20 17-38-49

**Comment by adborden** [comment]
Just for comparison, here's how that case comes up in the fec.gov archived MURs search http://www.fec.gov/MUR/
screenshot from 2016-09-20 17-47-01

**Comment by porta-antiporta** [comment]
@adborden yeah there are a handful of MURs with none of that metadata.  Not much we can do at this point given the lack of information.

**Comment by adborden** [comment]
Great, so it sounds like this case is worth handling. @jenniferthibault do you have some initial thoughts on how you want to handle the no metadata case? Should I create a new issue for you to track this?

**Comment by jenniferthibault** [comment]
Carried the empty fields issue over into #1598

**Pull request merged**